lib/v2/bnu/dwxgb.js

```diff
@@ -0,0 +1,57 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const { category, type } = ctx.params;
+
+    const rootUrl = 'http://dwxgb.bnu.edu.cn';
```

Comment:
```suggestion
    const rootUrl = 'https://dwxgb.bnu.edu.cn';
```
HTTP traffic will be [redirected](https://github.com/DIYgod/RSSHub/runs/8136463867?check_suite_focus=true#step:11:16) to HTTPS

Reply:
👌